recipes/sqsgenerator/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "sqsgenerator" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dnoeger/{{name}}/archive/{{version}}.tar.gz
+  sha256: b7550c645af75e5b50711c1c83a6ba4ef6461616cdf48f33cb42cb8893ce2c83
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [not linux or py<35]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - gmp 
+    - cython
+    - pymatgen
+  run:
+    - python
+    - gmp 
+    - cython
```

Comment:
does this really need cython at run time?

Reply:
You are right - it is not required at runtime - so I moved it to the `build` section.